webpack.config.js

```diff
@@ -1,38 +1,38 @@
+// The path to the CesiumJS source code
+const cesiumSource = 'node_modules/cesium/Source';
+const cesiumWorkers = '../Build/Cesium/Workers';
+const CopyWebpackPlugin = require('copy-webpack-plugin');
 const path = require('path');
-
 const webpack = require('webpack');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const CopyWebpackPlugin = require('copy-webpack-plugin');
 
-module.exports = [{
-    mode: 'development',
+module.exports = {
     context: __dirname,
     entry: {
         app: './src/index.js'
     },
     output: {
-        filename: '[name].js',
-        path: path.resolve(__dirname, 'dist')
+        filename: 'app.js',
+        path: path.resolve(__dirname, 'dist'),
+        sourcePrefix: ''
     },
-    devtool: 'eval',
-    node: {
-        // Resolve node module use of fs
-        fs: "empty",
-        Buffer: false,
-        http: "empty",
-        https: "empty",
-        zlib: "empty"
+    amd: {
+        // Enable webpack-friendly use of require in Cesium
+        toUrlUndefined: true
     },
     resolve: {
-        mainFields: ['module', 'main']
+        alias: {
+            // CesiumJS module name
+            cesium: path.resolve(__dirname, cesiumSource)
```

Comment:
You've defined the `cesium` alias here, but I don't see it used in `src/index.js`.

It's pretty atypically to define relative paths to dependencies in `node_modules` from source code like what's currently happening in `index.js`. Instead, you usually rely on the package name itself which [webpack resolves from `node_modules` just like node resolves](https://webpack.js.org/configuration/resolve/#resolvemodules).

At first, I would have suggested omitting the `resolve` property altogether, which would make the first line in `index.js`
```
import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "cesium";
```

However, there's an issue when importing `Widgets/widgets.css`. In regards to css files with cesium and webpack, there's actually [an issue in the CesiumJS](https://github.com/CesiumGS/cesium/issues/9212) repo advising the best configuration. Basically, we don't explicitly export anything other than JS files as part of the cesium npm package, which means webpack does not allow us to resolve CSS files like js modules.

After testing this out myself, I think we should keep the above alias, but configure the Cesium's main module file so that Cesium can be resolved without a relative path. So in `webpack.config.js`, `resolve` becomes

```
    resolve: {
        alias: {
            cesium: path.resolve(__dirname, cesiumSource)
        },
        mainFiles: ['module', 'main', 'Cesium']
    },
 ```
 and the first few lines of `index.js` become
 ```
import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "cesium";
import "cesium/Widgets/widgets.css";
```

Reply:
Thanks for the suggestion! I believe that is captured in the issue I linked to above. While that change is out of the scope of this PR and repo, we can adjust this example if/when Cesium's `package.json` file is updated.